docs/reference/mapping/dynamic/templates.asciidoc

```diff
@@ -124,18 +128,54 @@ PUT my-index-000001/_doc/1
 <1> The `my_integer` field is mapped as an `integer`.
 <2> The `my_string` field is mapped as a `text`, with a `keyword` <<multi-fields,multi field>>.
 
+[[match-mapping-runtime-fields]]
+===== Mapping runtime fields in a dynamic template
+You can also map runtime fields in a dynamic template. For example, the
+following request adds a dynamic template named `dayOfWeek` that maps all
+incoming `string` fields as `keyword` fields in the `runtime` section of the
+mapping. Although the `runtime` definition is blank, new `string` fields will
+be mapped as `keyword` runtime fields based on the <<dynamic-field-mapping-types,dynamic mapping rules>> that {es} uses for adding field types to the
+mapping. Any `string` that doesn't pass date detection or numeric detection is
+automatically mapped as a `keyword` when `dynamic` is set to `runtime`.
```

Comment:
The mention of dynamic runtime here is out of place. If you use dynamic: runtime, you normally would not need to define runtime fields from dynamic templates. If you use dynamic: true, then it would make sense to define some fields as runtime fields, as they would otherwise be indexed.

Reply:
Per our conversation, we'll handle this distinction in an upcoming PR.